geode-core/src/main/java/org/apache/geode/distributed/ServerLauncher.java

```diff
@@ -333,13 +333,15 @@ private ServerLauncher(final Builder builder) {
     controlHandler = new ControlNotificationHandler() {
       @Override
       public void handleStop() {
+        logger.debug("Handling ControllableProcess stop request.");
```

Comment:
@kirklund what's the value of these two new debug-level log messages?

Reply:
Honestly I think these two debug statements should be removed. They had some value while debugging acceptance tests in Java 17 but keeping it doesn't seem valuable.